packages/documentation/src/stories/misc/migration-guide/setup.component.ts

```diff
@@ -51,6 +52,22 @@ export class SetupComponent extends LitElement {
               your application.
             </div>
           </div>
+          <div class="col-12">
```

Comment:
This checkbox would fit better under the "Component Migration" of the "Migration from v9 to v10" section. The main filters are more about project setup, and at this point it’s not clear what "automatic migration" means.

Also, if the checkbox stays here, the filter should apply to all migration sections (e.g. "Migration from v4 to v5").